build.sbt

```diff
@@ -232,27 +232,23 @@ lazy val benchmarks = project
 
 lazy val docs = project
   .in(file("zio-prelude-docs"))
+  .settings(stdSettings("zio-prelude-docs"))
   .settings(
-    scalaVersion                               := BuildHelper.Scala213,
-    publish / skip                             := true,
-    moduleName                                 := "zio-prelude-docs",
     scalacOptions -= "-Yno-imports",
     scalacOptions -= "-Xfatal-warnings",
+    scalaVersion                               := Scala213,
+    crossScalaVersions -= Scala211,
+    projectName                                := "ZIO Prelude",
+    mainModuleName                             := (core.jvm / moduleName).value,
+    projectStage                               := ProjectStage.ProductionReady,
     ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(
-      core.jvm,
-      experimental.jvm,
-      experimentalLaws.jvm,
-      laws.jvm,
-      scalaParallelCollections
```

Comment:
Why remove these?

Reply:
I'm making some experimental changes to make ci passes. Finally, I can revert unrelated changes.